### PR TITLE
Remove custom documentation of PhpStorm Gerrit plugin in appendix

### PR DESCRIPTION
**Pull request by pixeldesu**
~~While the plugin can be set up, the setup is more convoluted and complex than the widely-used CLI setup that is documented pretty well in the contribution workflow anyway.~~

~~So, in order for no one else stumbling upon this, it's best to just remove it.~~

Not a lot of people in/around the Core circles use the PhpStorm Gerrit plugin. We noticed that there can be some issues with setting it up (depending on how deep you want the integration to go). As the plugin is still useful to some we're not fully removing any mention of it, but instead just link to the Marketplace without providing our own documentation around the plugin usage.

### CODE REVIEW

**Comment by garvinhicking** [comment]
I agree.

We should also remove the page itself, to prevent having an orphan entry (mobile atm, can't add that to the PR)

**Comment by liayn** [comment]
IMO the existence of the plugin should still be mentioned.
It is super helpful when pushing stuff.
(I do not use any other feature of it, actually)

**Comment by garvinhicking** [comment]
We could then only link to that plugin via marketplace URL https://plugins.jetbrains.com/plugin/7272-gerrit just like the TYPO3 plugin, and not give any special instructions which would make it our responsibility to describe it more properly?

**Comment by pixeldesu** [comment]
@garvinhicking added the link to the Marketplace now!

**Comment by garvinhicking** [comment]
@pixeldesu Thanks! Since I'm only on mobile, could you do me two more favors (or wait a bit until I can make it):

* Add the old anchor `.. _phpstorm-gerritplugin:` before the gerrit marketplace link, so that references to that section at least resolve to the new overview and not break possible docs pointing towards it
* Also remove the referenced gerritplugin images referenced in the deleted detail section, since those are not used anymore

:-)

**Comment by pixeldesu** [comment]
Done!

**Review by linawolf — APPROVED** [review]
Good idea, thank you

File: Documentation/Appendix/IdePhpStormSetup.rst [hunk 1]
```diff
@@ -132,14 +132,17 @@ Recommended Plugins
...
+
 Optional Plugins
 ----------------
 
 * `Fluid plugin <https://plugins.jetbrains.com/plugin/11151-fluid--enterprise>`__
   (sgalinski)
 * `TypoScript plugin
   <https://plugins.jetbrains.com/plugin/11243-typoscript--enterprise>`__ (sgalinski)
-* :ref:`phpstorm-gerritplugin`
+* `Gerrit plugin
```
Comment (garvinhicking) [inline Documentation/Appendix/IdePhpStormSetup.rst:144]:
```suggestion
 * `Gerrit plugin
```

**Review by garvinhicking — DISMISSED** [review]

**Review by garvinhicking — APPROVED** [review]

**Comment by garvinhicking** [comment]
Thanks!

**Pull request merged**